content/articles/using-django-in-kivy-application/index.md

```diff
@@ -0,0 +1,596 @@
+    def __init__(self, **kwargs):
+        super(AddScreen, self).__init__(**kwargs)
+        self.box = BoxLayout()
+        self.box.orientation = "vertical"
+        self.box.add_widget(Label(text="Add To List...", color="blue",pos_hint={"top": 1}))
+        self.addNewForm = AddNewForm()
+        self.box.add_widget(self.addNewForm)
+        self.add_widget(self.box)
+#Screen Management
+class ScreenManagement(ScreenManager):
+    pass
+#app class
+class TodoApp(App):
+    pass
+
+if __name__ == '__main__':
+    TodoApp().run()
+```
+Let us also edit the `todo.kv ` to:
+
+`your folder/todo.kv`
+```kv
+
+BoxLayout:
+    orientation: 'vertical'
+    Menu:
+        size_hint_y: .1
+        manager: screen_manager
+    ScreenManagement:
+        size_hint_y: .9
+        id: screen_manager
+
+
+
+<Menu>:
+    orientation: "vertical"
+    ActionBar:
+        top: 0
+        ActionView:
+            ActionPrevious:
+            ActionButton:
+                text: 'Home'
+                on_press: root.manager.current = 'screen_home'
+
+            ActionButton:
+                text: 'Add New'
+                on_press: root.manager.current = 'screen_add'
+<ScreenManagement>:
+    id: screen_manager
+    HomeScreen:
+        name: 'screen_home'
+        manager: 'screen_manager'
+    AddScreen:
+        name: 'screen_add'
+        manager: 'screen_manager'
+<HomeScreen>:
+    BoxLayout:
+        orientation: "vertical"
+        MyRecycleView:
+<MyRecycleView>:
+    viewclass: 'Label'
+    RecycleBoxLayout:
+        color: 1,1,1,1
+        default_size: None, dp(56)
+        default_size_hint: 1, None
+        size_hint_y: None
+        height: self.minimum_height
+        orientation: 'vertical'
+<AddNewForm>:
+    text_input: input
+    TextInput:
+        id: input
+        pos: root.center_x - 220, 300
+        size: 400,50
+    Button:
+        size: 130,40
+        pos: root.center_x - 100, 200
+        text: 'Submit'
+        on_release: root.submit_input()
+```
+The `AddNewForm` contains a function `submit_input` that makes a post request to our server passing the input text as data.
+The form has a textinput that one can type the task, and a button that calls the `submit_input` function when released.
+We then declare a `BoxLayout ` class that will contain the `AddNewForm` and a label text "Add to list..."
+
+When you now click `Add New` you should now see the following:
+
+![Creating a task](section-engineering-education/engineering-education/content/articles/using-django-in-kivy-application/creating_a_task.png)
+
+When you create a task, say "Complete application" and click submit button once and click `Home`, you should have the following:
+
+![Task submitted successfully](section-engineering-education/engineering-education/content/articles/using-django-in-kivy-application/task_submitted_successfully.png)
+
+### Conclusion. 
+We have learnt how create a kivy application and how to use Django in such applications. You can therefore
```

Comment:
Conclusion could have been more elaborate.

Give a summary of what was covered.

Add a topic `Further reading` that lists out various reference links